lib/Data/Validate/Sanctions.pm

```diff
@@ -322,8 +333,8 @@ sub _index_data {
 
     $self->{_index} = {};
     for my $source (keys $self->{_data}->%*) {
-        my @content = ($self->{_data}->{$source}->{content} // [])->@*;
-        warn "Content is empty for the sanction source $source. The sanctions file should be updated." unless @content;
+        my @content = clone($self->{_data}->{$source}->{content} // [])->@*;
```

Comment:
why `clone`? `@content` will be an independent copy even without it.

Reply:
No. The elemets of `content` are hash-references with new attributed in the following lines of code. I decided to clone them to avoid side effects, especially on tests.